### PR TITLE
Fix formatting of matlab rst

### PR DESCRIPTION
**Pull request by pwalczysko**
Some formatting issues fixed here.

@dominikl - Do you know what is the 

``The Image Processing toolbox is only necessary...`` remark about ? Indeed, it did not seem necessary and I wonder why is the sentence there ?

### CODE REVIEW

File: docs/matlab.rst [hunk 1]
```diff
@@ -33,21 +27,17 @@ We will show:
 
    - To install the OMERO.matlab toolbox https://www.openmicroscopy.org/omero/downloads/
 
-   - The Image Processing toolbox is only necessary for the images
-      analysis. This is a convenient toolbox for analysis purpose.
-      You do not need to install that toolbox to integrate OMERO and
-      MATLAB.
+   - The Image Processing toolbox is only necessary for the image analysis. This is a convenient toolbox for analysis purpose. You do not need to install that toolbox to integrate OMERO and MATLAB.
```
Comment (jburel) [inline docs/matlab.rst]: you will need it for the segmentation part

Comment (pwalczysko) [reply docs/matlab.rst]: In that case, it is not clear to me where did I install it. 
My Matlab works fine with the whole workflow, getting the segmentation and the ROIs back to OMERO. But I am not aware of installing any Image Processing toolbox. This is not clear to me... @dominikl

Comment (dominikl) [reply docs/matlab.rst]: Then you probably have it already installed. Check your preferences?
Screen Shot 2020-03-02 at 09 35 34

Comment (dominikl) [reply docs/matlab.rst]: But yes, maybe we could add a link to matlab, to make it clearer what this is about:
https://uk.mathworks.com/products/image.html

Comment (pwalczysko) [reply docs/matlab.rst]: @dominikl yes, indeed, as checked in Preferences, I have it already installed.

Did Matlab install it for me on the strenght of my account history ? I re-installed Matlab on this machine just couple of months ago. Yes, I will have a look about adding that link @dominikl is pointing to.

Comment (dominikl) [reply docs/matlab.rst]: I guess so. It probably checks if it's part of your license and then installs it automatically. Unfortunately I can't remember how I installed it in first place.

**Comment by dominikl** [comment]
Built the docs locally, looks good. Had a quick run through the script in matlab, works fine. Good to merge from my point of view 👍

**Pull request merged**